go/kbfs/libkbfs/chat_rpc.go

```diff
@@ -747,6 +753,8 @@ func (c *ChatRPC) NewChatActivity(
 		}
 
 		if len(cbs) == 0 {
+			c.config.GetPerfLog().CDebugf(ctx,
+				"newNotificationChannel ChatActivityType_INCOMING_MESSAGE")
```

Comment:
@songgao i'm nervous this may be called too often. We really don't want to overwhelm the perf log with calls since it will rotate faster and not be as useful to find anomalies.

Reply:
If I understand correctly, this only gets called if we haven't listened on this conversation yet (i.e. `len(cbs) == 0`). Not 100% sure though. But I guess this is sort of redundant since we can just infer this if `GetFavorites newNotificationChannel` shows up and `newNotificationChannel ChatActivityType_NEW_CONVERSATION` doesn't.